python/pyspark/sql/functions.py

```diff
@@ -11157,6 +11157,142 @@ def udf(
         return _create_py_udf(f=f, returnType=returnType, useArrow=useArrow)
 
 
+# ---------------  Conditional functions ------------------------
+
```

Comment:
`if`: new add on the scala side, however it conflicts with Python keywords `if` and cannot be added on the Python side?

Reply:
I think we should apply a different name in this case, like `map` in Scala <-> `create_map` in Python.

cc @HyukjinKwon @beliefer do you have a good name for it?

Reply:
friendly ping @HyukjinKwon  for the function names for `any` and `if` ?

Reply:
@beliefer @panbingkun sorry, but I'd suggest we exclude `any` `if` for now: we have `some/bool_or` to replace `any`, and `when/otherwise` to replace `if`.

let's add them in separate PRs later if we really need them

Reply:
ok, let me remove it.